src/thunkMiddleware.js

```diff
@@ -1,4 +1,7 @@
-export default ({ dispatch, getState }) => next => async action => {
+export default parseError => ({
```

Comment:
Maybe this should be a different middleware.

Reply:
I agree with this too. It could add more flexibility if this is a different middleware. But is not mandatory